circle.yml

```diff
@@ -0,0 +1,14 @@
+dependencies:
+  override:
+    - pip install awscli==1.11.139
+
+test:
+  override:
+    - test 0 = 0
+
+deployment:
```

Comment:
yep sorry saw it too late, I'll do the change in another pr, let's see if it actually works first.